fields/types/list/ListType.js

```diff
@@ -185,17 +185,19 @@ list.prototype.updateItem = function (item, data, files, callback) {
 
 	var field = this;
 	var values = this.getValueFromData(data);
-	// Don't update the value when it is undefined
-	if (values === undefined) {
-		return utils.defer(callback);
-	}
-	// Reset the value when null or an empty string is provided
-	if (values === null || values === '') {
+	// Reset the value when no value or null or an empty string is provided
+	if (values === undefined || values === null || values === '') {
 		values = [];
 	}
-	// Wrap non-array values in an array
 	if (!Array.isArray(values)) {
-		values = [values];
+		if (typeof values === 'object' && values[0] !== undefined) {
```

Comment:
I think you should use `hasOwnProperty` in this instance, although for this particular use case, I suppose array values should always be at least `null`? Not sure.